pkg/commands/executor.go

```diff
@@ -97,19 +107,8 @@ func NewExecutor(buildInfo BuildInfo) *Executor {
 		}
 	}
 
-	// init of commands must be done before config file reading because
-	// init sets config with the default values of flags
-	e.initRoot()
-	e.initRun()
-	e.initHelp()
-	e.initLinters()
-	e.initConfig()
-	e.initVersion()
-	e.initCache()
-
-	// init e.cfg by values from config: flags parse will see these values
-	// like the default ones. It will overwrite them only if the same option
-	// is found in command-line: it's ok, command-line has higher priority.
+	// init e.cfg by values from config: flags parse will see these values like the default ones.
+	// It will overwrite them only if the same option is found in command-line: it's ok, command-line has higher priority.
```

Comment:
Yes, I follow https://sembr.org/ because for me the number of character limit is coming from the past, I think we already discussed that inside an issue or PR about `lll` :smile:

Reply:
I don't know if we should recommend that but I share the opinion of the article you provided.
For me, readability is the key, I often reformat a comment only when I try to read it and it feels painful to read.